tests/dialects/test_mysql.py

```diff
@@ -581,6 +579,25 @@ def test_mysql_time(self):
         self.validate_all(
             "STR_TO_DATE(x, '%Y-%m-%dT%T')", write={"presto": "DATE_PARSE(x, '%Y-%m-%dT%T')"}
         )
+        self.validate_all(
+            "SELECT FROM_UNIXTIME(col)",
+            read={
+                "postgres": "SELECT TO_TIMESTAMP(col)",
+            },
+            write={
+                "mysql": "SELECT FROM_UNIXTIME(col)",
+                "postgres": "SELECT TO_TIMESTAMP(col)",
+                "redshift": "SELECT (TIMESTAMP 'epoch' + col * INTERVAL '1 SECOND')",
+            },
+        )
+        self.validate_all(
+            "SELECT FROM_UNIXTIME(1711366265, '%Y %D %M')",
+            write={
+                "mysql": "SELECT FROM_UNIXTIME(1711366265, '%Y %D %M')",
+                "postgres": "SELECT TO_TIMESTAMP(1711366265)",
+                "redshift": "SELECT (TIMESTAMP 'epoch' + 1711366265 * INTERVAL '1 SECOND')",
+            },
+        )
```

Comment:
Since the MySQL query doesn't really map well to postgres and redshift, can we just make this an identity test to avoid future confusion?

MySQL
```
mysql> SELECT FROM_UNIXTIME(1711366265, '%Y %D %M');
+---------------------------------------+
| FROM_UNIXTIME(1711366265, '%Y %D %M') |
+---------------------------------------+
| 2024 25th March                       |
+---------------------------------------+
1 row in set (0.00 sec)
```

Postgres:
```
georgesittas=# SELECT TO_TIMESTAMP(1711366265);
      to_timestamp
------------------------
 2024-03-25 13:31:05+02
(1 row)
```

Redshift:
```
SELECT (TIMESTAMP 'epoch' + 1711366265 * INTERVAL '1 SECOND')
-- ?column?
-- 2024-03-25 11:31:05
```

Reply:
Was debating whether it's better to use `validate_all` in order to show that the `format` arg is dropped in transpile or to just use an identity check, will revert to the latter

Reply:
Yeah, we should avoid creating tests like that because they're unstable and aren't really useful. If we want to put in the work and handle the unsupported arguments properly, we'd need to just do `self.unsupported` and check for an `UnsupportedError` in the validate all call.